conf/application.conf

```diff
@@ -95,7 +95,7 @@ metrics {
 }
 
 assets {
-  version = "3.4.0"
+  version = "3.15.0"
   version = ${?ASSETS_FRONTEND_VERSION}
```

Comment:
This line can be removed probably. The same with `url = ${?ASSETS_URL}`

We're defining those two values twice, I can imagine that it's related with some global variables, but not sure if we need to have defined it twice

Reply:
I've always wondered about these definitions.  I was this - https://github.com/hmrc/assets-frontend#frontends
I guess our service was based on init-service originally?   Best leave it for now?

Reply:
Yes, we can leave it for now